### PR TITLE
allow sessions to file. fixes #2001, #2186 and refs #3454

### LINKED ISSUE #2001: Storing Sessions in File Not Working
When a site is configured to store sessions in file (instead of a database), the sessions do not get get written to the file.

/index.php?module=security-center&type=admin&func=modifyconfig

### PR DESCRIPTION
**Pull request by craigh**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | fixes #2001, #2186 
| Refs tickets      | #3454
| License           | MIT
| Changelog updated | yes

### CODE REVIEW

**Comment by craigh** [comment]
hmm - a problem with the CLI invalid csrf ... 🤔 😠 

@matthiasnoback - do you know why symfony-console-form is using the tokens? I thought it would ignore them? see https://travis-ci.org/zikula/core/builds/219052481

**Comment by craigh** [comment]
well I canme up with a sort of hacky fix to the csrf problem when using cli.

File: src/docs/Core-2.0/Session.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Session Information
+===================
+
+Sessions can be stored in the Database or in the Filesystem. This choice as well as a few configuration options
+can be set in the ZikulaSecurityModule:
```
Comment (Guite) [inline src/docs/Core-2.0/Session.md]: Typo. How about `Zikula SecurityCenter module`?

File: src/lib/Zikula/Bundle/CoreInstallerBundle/Stage/Install/CompleteStage.php [hunk 2]
```diff
@@ -60,14 +60,19 @@ public function getTemplateParams()
 
     public function getResponse(Request $request)
     {
+        $router = $this->container->get('router');
         if ($this->sendEmailToAdmin($request)) {
             $request->getSession()->getFlashBag()->add('success', $this->__('Congratulations! Zikula has been successfully installed.'));
+            $request->getSession()->getFlashBag()->add('info', $this->__f(
+                'Session are currently configured to use the filesystem. It is recommended that you change this to use the database. Click %here% to configure.',
+                ['%here%' => '<a href="' . $router->generate('zikulasecuritycentermodule_config_config') . '">' . $this->__('Security Center') . '</a>']
+            ));
```
Comment (Guite) [inline src/lib/Zikula/Bundle/CoreInstallerBundle/Stage/Install/CompleteStage.php]: Would it be possible to change it to the database automatically during/after the `CompleteStage`?

**Comment by matthiasnoback** [comment]
@craigh Indeed, I thought the `csrf_protection` was always set to `false` when using SymfonyConsoleForm. Maybe a newer form version has broken it? Don't know.

File: src/lib/Zikula/Bundle/CoreInstallerBundle/Stage/Install/CompleteStage.php (hunk 2 above)
Comment (craigh) [inline src/lib/Zikula/Bundle/CoreInstallerBundle/Stage/Install/CompleteStage.php]: it forces a re-login, so I didn't think that was the best method.

**Comment by craigh** [comment]
@matthiasnoback - we are using an older version of your library with this branch - could that be the cause? Was there anything special with the form classes I was to do?

**Comment by matthiasnoback** [comment]
@craigh Honestly, don't know.

**Comment by craigh** [comment]
@matthiasnoback looks like I just needed to update the library. I was using an older version (v1) because this branch originally was restricted to php >= 5.4, but now it is updated to require php >=5.5 and so I can use v2 of the lib and this fixes the problem.

**Pull request merged**